.github/workflows/lint-test.yml

```diff
@@ -24,17 +24,17 @@ on:
       - "**"
 jobs:
   lint-test:
-    runs-on: ubuntu-20.04
```

Comment:
This should be pinned. It should be pinned to a sha, in fact, to ensure reproducibility.

Reply:
I was running the actions locally with act and this was mainly just to get it running at all. Easy change and you are of course right